libs/@guardian/source-react-components-development-kitchen/src/footer-with-contents/FooterLinks.tsx

```diff
@@ -19,7 +20,7 @@ type FooterLink = AnchorHTMLAttributes<HTMLAnchorElement> & {
 	/**
 	 * The URL of the link
 	 */
-	href: string;
+	href?: string;
```

Comment:
Given this is exclusively for links - and we use `<button>`s for non-`href`ed elements, why is this optional?

Reply:
This would also make the [type-guard](https://github.com/guardian/csnx/pull/940/files#diff-4e786fe5e22a0794f76d2b114359a2cd833c90407597c3a55e6e8417107ec951R82) not completely true, because `FooterLink`s can have `.href === undefined`, so in essence the guardian and type don't match.

Reply:
@jamesgorrie @rBangay this doesn't need to be optional, I'd switched it to optional when I was initially looking at the types to see if I could / should use the same type, which I opted not to do, so yes I just forgoto to undo this change! Will restore as non-optional now.